eos-updater-flatpak-installer/tests/Makefile.am

```diff
@@ -0,0 +1,63 @@
+# Copyright © 2017 Endless Mobile, Inc.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+include $(top_srcdir)/glib-tap.mk
+
+installed_testdir = $(libexecdir)/installed-tests/libeos-updater-flatpak-installer-@EUU_API_VERSION@
```

Comment:
Drop `-@EUU_API_VERSION@` from these lines (and, indeed, anything to do with libeos-updater-flatpak-installer). `EUU_API_VERSION` is the API version of libeos-updater-util (EUU is its acronym). Technically we should have an `EUFI_API_VERSION` for libeos-updater-flatpak-installer, but I think that’s got too high a pain-to-benefit ratio to ask you to do in this PR. I’ll do it later.

Reply:
Yeah, I wasn't sure about whether or not we wanted to version them in lockstep. For now I'll just create a EUFI_API_VERSION and set it to 0